packages/vuetify/src/services/theme/__tests__/theme.spec.ts

```diff
@@ -127,7 +125,7 @@ describe('Theme.ts', () => {
     expect(spy).toHaveBeenCalledTimes(1)
   })
 
-  it('should use themeCache', () => {
+  it.skip('should use themeCache', () => {
```

Comment:
mistake, broken, or not relevant anymore?

Reply:
Test broke and it's a useless feature so I couldn't be bothered

Reply:
Fixed, the original test was incorrect